src/interfaces/WF/WFWorkflowActionParameters.ts

```diff
@@ -61,6 +61,7 @@ interface WFWorkflowActionParameters {
   WFDontIncludeFileExtension?: boolean;
   WFEncodeMode?: WFEncodeMode;
   WFFlashlightSetting?: WFFlashlightSetting;
+  WFFrequency?: string[];
```

Comment:
Could this be changed to `DayOfWeek[]` please? 🙂

Reply:
The frequency can also be the `Ask When Run`, and I believe the structure is

```json
"WFItemSpecifier": {
  "Value": {
    "Type": "Ask"
  },
  "WFSerializationType": "WFTextTokenAttachment"
}
```

There's a iOS client bug where this value isn't persisted when set when using `Ask When Run` in the create alarm action.  Try setting the frequency to `Ask When Run` in app, Clicking `Done`, then going back into the shortcut editor. It will be reset to `Never`.

This should still be implemented at some point, but more thought will have to go into how to handle how many different ways `Ask When Run` is used (list item, inline variable, magic variable, etc)

Reply:
After the discussion in the other comment it will now need to be changed to:

```ts
WFFrequency?: WFFrequency[];
```